web/modules/custom/dpl_fees/src/Form/FeesListSettingsForm.php

```diff
@@ -120,6 +120,28 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#default_value' => $config->get('payment_site_button_label'),
     ];
 
+    $form['settings']['blocked_user'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Blocked user', [], ['context' => 'Library Agency Configuration']),
+      '#collapsible' => FALSE,
+      '#collapsed' => FALSE,
+    ];
+
+    $form['settings']['blocked_user']['blocked_patron_e_link_url'] = [
+      '#type' => 'linkit',
+      '#title' => $this->t('Blocked user link for modal', [], ['context' => 'Library Agency Configuration']),
+      '#description' => $this->t('If a user is blocked because of fees a modal appears. This field makes it possible to place a link in the modal to e.g. payment options or help page. <br>
+                                         If left empty, the link will not be shown. <br>
+                                         You can add a relative url (e.g. /takster). <br>
+                                         You can search for an internal url. <br>
+                                         You can add an external url (starting with "http://" or "https://").', [], ['context' => 'Library Agency Configuration']),
```

Comment:
I would suggest changing the context of the translatable strings to match the rest of the form. 

So change "Library Agency Configuration" --> "Fee list settings form".